change/@azure-msal-browser-2020-11-06-14-12-06-update-request-types.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

Comment:
```suggestion
  "type": "minor",
```

Reply:
Why? API surface isnt changing.

Reply:
Removing the export of `AuthorizationUrlRequest` may be worth making this a minor update

Reply:
Will remain patch type